packages/shared/src/components/banners/FixedBottomBanner.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { ReactElement } from 'react';
+import CloseButton from '../CloseButton';
+import LoginButton from '../LoginButton';
+import { Button } from '../buttons/Button';
+import { tellMeWhy } from '../../lib/constants';
+
+interface FixedBottomBannerProps {
+  onDismiss(): void;
+  daysLeft: number;
+}
+
+export function FixedBottomBanner({
+  onDismiss,
+  daysLeft,
+}: FixedBottomBannerProps): ReactElement {
+  return (
+    <div className="flex fixed inset-0 z-max justify-center items-center p-6 w-full bg-gradient-to-l top-[unset] from-[#EF43FD] to-[#6451F3]">
+      <CloseButton
+        className="top-4 right-4"
+        position="absolute"
+        onClick={onDismiss}
+      />
+      <div className="flex flex-col laptop:flex-row gap-4 justify-center laptop:items-center w-full laptop:max-w-[52.25rem]">
+        <div className="flex flex-col flex-1 gap-2 w-full">
+          <h1 className="font-bold typo-large-title">
+            Registration is going to be required in {daysLeft} day
+            {daysLeft > 1 ? 's' : ''}
+          </h1>
+          <p className="flex flex-1 w-full typo-title3">
+            We’d love to see you join our community ❤︎. Signing up gets you free
+            access to the personalized feed, discussions, squads, AI search, and
+            more!
+          </p>
+        </div>
+        <div className="flex flex-row laptop:flex-col gap-2">
+          <LoginButton
+            showLoginButton={false}
+            copy={{ signup: 'Sign up now' }}
+          />
```

Comment:
I thought the idea was it simply linked to `/onboarding` no?

Reply:
Yeah described in specs as well:


“Sign up” | If the user clicks on sign up, we open the onboarding v3 or v2 depending on the experimentation bucket
-- | --


So you can just use a normal button and link it hardcoded to webapp onboarding.
Then maybe revert changes on loginButton.
https://dailydotdev.atlassian.net/wiki/spaces/HAN/pages/614432779/Anon+users+sign+up+changes+Login+page+for+extension

Also good side point would be:
This makes a perfect example of verify before building as we discussed about.

Reply:
Oh, there was no link to the spec in the ticket. I only based on the JIRA ticket which did not mention the behavior of the button itself.

Reply:
Let me read the spec now and see if there are any differences with the current implementation.